backend/processing/check_files_checksums/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt update && apt install -y python3
+COPY task.py /opt/task.py
+RUN chmod +x /opt/task.py
```

Comment:
If you make task.py executable *locally* you shouldn't need to run this.

Reply:
Do you mean assigning executable permissions for script in source code ? I don't like this approach.

Reply:
I like more approach of setting it during deployment exactly how it should be (setting executable bit for u, g, o) with correct permissions as well.

Reply:
Can we just leave it as is please ?